src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,126 @@
+// ----- BASE ELEMENTS ---------------
+
+.eyebrow {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
+}
+
+.img-left {
+  float: left;
+  width: 159px;
+  height: 56px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.right-menu {
+  float: right;
+}
+
+.eyebrow a img {
+  width: 119px;
+  height: 24px;
+}
+
+.eyebrow a {
+  float: left;
+  font-size: 13px;
+  color: #ffffff;
+  text-align: center;
+  padding: 1rem 9px;
+  text-decoration: none;
+  line-height: 1;
+  height: 56px;
+}
+
+.eyebrow p {
```

Comment:
Please change to `p {`.

Also, please move under `hxTopnav-primary` class scope:

Reply:
Changes have been done, Please review